tests/functional/README-functional.md

```diff
@@ -0,0 +1,139 @@
+# Functional Tests
+
+- [Functional Tests](#functional-tests)
+  - [Summary](#summary)
+  - [Rational](#rational)
+  - [Testing Environment](#testing-environment)
+  - [Submitting Tests](#submitting-tests)
+    - [Understanding the main steps of CytoSnake's Functional Tests](#understanding-the-main-steps-of-cytosnakes-functional-tests)
+      - [Preparation](#preparation)
+      - [Execution](#execution)
+    - [Output Check](#output-check)
+  - [Ending Remarking](#ending-remarking)
+
+## Summary
+
+Here, we provide you with `CytoSnake` functional test documentation.
+It is designed to help you, as both a developer and user, gain a comprehensive understanding of the test cases and their expected outcomes.
+By carefully reviewing this documentation, you can efficiently perform functional tests, validate `CytoSnake`'s behavior, and ensure its reliability in various scenarios.
+
+## Rational
+
+Functional tests are essential for evaluating `CytoSnake`'s behavior in accurately executing its intended functions with specific input parameters.
+They primarily aim to validate the functionality of the program, ensuring it meets the specified requirements.
+These tests involve feeding `CytoSnake`'s Command Line Interface (CLI) with various parameters to simulate user interactions and thoroughly monitor the user experience
+By conducting functional tests, developers can ensure that all user-inputted parameters function correctly within `CytoSnake`, leading to a more reliable and user-friendly application.
+Ultimately, the focus of these tests is to guarantee easy and simple user experience with `CytoSnake`.
+
+## Testing Environment
+
+Below is an image on how the testing environment:
```

Comment:
```suggestion
```

I think this is redundant with the figure legend and title, but maybe there is key detail there that I'm missing.